tests/alive-tv/memory/memcmp-noub3.srctgt.ll

```diff
@@ -0,0 +1,23 @@
+target datalayout = "e-p:64:64:64"
+
+define i32 @src() {
```

Comment:
isn't this test exactly equal to the previous one? Maybe you wanted to return poison or smth?

Reply:
Oh yeah, it was my mistake.
It was supposed to be the removed ub2 function.